charts/mgr-tenants/values.yaml

```diff
@@ -0,0 +1,159 @@
+# Default values for mgr-tenants.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: 732722833398.dkr.ecr.us-west-2.amazonaws.com732722833398.dkr.ecr.us-west-2.amazonaws.com/mgr-tenants
+  pullPolicy: IfNotPresent
```

Comment:
It would be nice to use "Always" here instead of "IfNotPresent".
We have "latest" tag for docker image so need to re-deploy every time to use the latest changes.